Wox.Infrastructure/StringMatcher.cs

```diff
@@ -19,7 +19,7 @@ public StringMatcher(IAlphabet alphabet = null)
             _alphabet = alphabet;
         }
 
-        public static StringMatcher Instance { get; internal set; }
+        public static StringMatcher Instance { get; set; }
```

Comment:
What needs to set this outside of the assembly?

Reply:
This same type of change is made all over.  Can you just explain to me why this is needed?  I might have not gotten that far in the review yet, but would like to understand it.

Reply:
Thanks for pointing it out, as it turn out I had deleted the file that makes these internals visible to others an had to strip them out. But I'm putting them back in place.